src/layout/section/landingGridSection/LandingGridSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { HeroSection, HeroSectionProps } from '../heroSection'
+import { LandingGridSectionElements } from './LandingGridSection.style'
+
+export type LandingGridSectionProps = HeroSectionProps &
+  Readonly<{
+    children: React.ReactNode
+  }>
+
+export const LandingGridSection = ({
+  children,
+  heroText,
+  heroImageUrlLarge,
+  heroImageUrlSmall,
+}: LandingGridSectionProps) => (
+  <LandingGridSectionElements.Grid>
```

Comment:
I'm still not convinced by this API with nested children items.
How does this work if I write that in kairos:
```
<LandingGridSectionElements.Grid>
    <LandingGridSectionElements.Figure>
      fist figure
    </LandingGridSectionElements.Figure>
    <LandingGridSectionElements.Body>{children}</LandingGridSectionElements.Body>
    <LandingGridSectionElements.Body>stuff</LandingGridSectionElements.Body>
    <LandingGridSectionElements.Figure>
      <HeroSection
        heroText={heroText}
        heroImageUrlSmall={heroImageUrlSmall}
        heroImageUrlLarge={heroImageUrlLarge}
        bottomElement={null}
      />
    </LandingGridSectionElements.Figure>
  </LandingGridSectionElements.Grid>
```

Reply:
You should not use `LandingGridSectionElements`in Kairos

Reply:
> You should not use LandingGridSectionElementsin Kairos

Are they exported?

Reply:
@guillaumewuip  Unfortunatelly. Is there a way to use it inside on component.tsx file without exporting it from style.tsx?

Reply:
@grosbouddha  @guillaumewuip  

Resolved here: [index.tsx](https://github.com/blablacar/ui-library/blob/b7558ae466b9c9e7cd6ff378aa0022bde23129ea/src/layout/section/SEALandingLayout/index.tsx#L1)